### DISCUSSION TITLE
Identity provider not showing up on login page

### DISCUSSION THREAD

**Discussion by arunrajac** [zitadel/zitadel#7155] [Q&A] [answered]
i added github identify provide but not reflecting.
can you please guid me.

**Comment by hifabienne** [comment]
Can you please add some more information?
Where did you add the identity provider?
On an organization or on the instance level?
If on an organization is it on the default organization or not?
Did you also activate the provider (should show a check icon)

**Comment by Madhusudan-vb** [comment]
I have added under the Profile settings as given below screenshot and it is activated. But still not seeing those IDP in our login page:
[cid:bccddf93-2c61-4824-b8ab-1dfd036c9852]



Thanks and Regards,
Madhusudan VB
________________________________

**Reply by hifabienne** [reply]
I can't see your screenshot

**Comment by Madhusudan-vb** [comment]
This is very urgent for us. Please reply with an solutin asap.

________________________________

**Reply by hifabienne** [reply]
Hei @arunrajac I just want to note here. That the discord chat is a community support channel and also with github questions and issues we give our best to help.
If you do have urgent needs for support and rely on the help of our team, please have a look at our enterprise support. https://zitadel.com/pricing

**Comment by arunrajac** [comment]
<img width="1066" alt="Screenshot 2024-01-08 at 3 10 08 PM" src="https://github.com/zitadel/zitadel/assets/142791868/093295b1-709c-490d-ba14-414a678571ef">
<img width="1063" alt="Screenshot 2024-01-08 at 3 10 20 PM" src="https://github.com/zitadel/zitadel/assets/142791868/9c3d95fa-58b3-47e2-ac61-fa3c54d1cf78">
<img width="730" alt="Screenshot 2024-01-08 at 3 10 37 PM" src="https://github.com/zitadel/zitadel/assets/142791868/2cdc720d-6f62-43c1-a557-0a4af25bce94">

 Settings
Modified login policy in the menu "Login Behavior and Security"
Enabled the attribute "External IDP allowed"
but provider not visible

**Reply by hifabienne** [reply]
Can you please also answer the following questions?

Where did you add the identity provider?
On an organization or on the instance level?
If on an organization is it on the default organization or not?
How does your authorize request look like?

**Reply by arunrajac** [reply]
Where did you add the identity provider?
-- under settings -> identity provider added two github and google.

On an organization or on the instance level?
-- an organization.

If on an organization is it on the default organization or not?
-- default organization.

How does your authorize request look like?
--redirecting link (zitadel login page)

**Comment by arunrajac** [comment]
@hifabienne can you please help me here. thanks

**Comment by Madhusudan-vb** [comment]
Hi Fabi,


I have added the the identity providers under the settings > Identity providers > Google, Microsoft. I have added the below screenshot for your reference.
[cid:920a0cae-66bc-4587-bd36-a08398bf72eb]



Thanks and Regards,
Madhusudan VB
________________________________

**Reply by hifabienne** [reply]
I can't see your screenshot again.

But let me try to explain what is important to know:
You can add the settings either as default values for your instance. you can see that on the top right: 
<img width="455" alt="image" src="https://github.com/zitadel/zitadel/assets/38692350/684d6214-2006-4b36-904d-6182ea2d1273">

Or you can configure them by adding it on a specific organization: 
<img width="455" alt="image" src="https://github.com/zitadel/zitadel/assets/38692350/4ba88f56-8687-42ab-96c0-05bd2be1f31a">

When it is for a specific organization I also want to know if that is your default organization or not, you can see that when going to your instance, organizations on the list the default is marked:
<img width="1249" alt="image" src="https://github.com/zitadel/zitadel/assets/38692350/5217db69-3e1f-423a-93a6-4a96f8ab96b0">

So where did you configure the settings?

**Reply by mffap** [reply, marked as answer]
@arunrajac based on our discussion: 

You don't have the appropriate Manager role to set the default organization. Ask your IAM Manager to mark the "buzz" org as default organization on the Instance.

The setup was confirmed working by sending the organization scope along to ZITADEL triggering the "buzz" organization. You can try that out by entering the information on our [Playground](https://zitadel.com/docs/apis/openidoauth/authrequest). Make sure to include the scope to trigger an organization: https://zitadel.com/docs/apis/openidoauth/authrequest#organization-policies-and-branding

How to move this to production?
a) You can set the default organization on the instance configuration to always load up the configuration of the organization "buzz"
b) In you application ("portal") include the org scope with the id of the "buzz" org into the SSO configuration. Sending the scope along will ensure that this organization will be selected.

**Comment by Madhusudan-vb** [comment]
I have an question: From Zitadel when I logged in trough SSO identity providers It's still asking for 2 factor authentication. How do we bypass this. But I don't want to delete or disable that 2 factor authentication because Incase if my SSO login not worked for a reason sometimes, So that I can use to login normally by non-idp user with 2 factor authentication. So is there a way to "Bypass" the 2 factor authentication Only if I logging in trough Identity Providers. Please someone help me here with an soultion.
